scripts/fuzz_opt.py

```diff
@@ -350,10 +350,20 @@ def is_git_repo():
     'typed_continuations_contnew.wast',
     'typed_continuations_contbind.wast',
     'typed_continuations_suspend.wast',
-    # New EH implementation is in progress
+    # TODO: make sure the fuzzer supports the new EH
+    'coalesce-locals-eh.wast',
+    'code-folding-eh.wast',
+    'code-pushing-eh.wast',
+    'dce-eh.wast',
+    'eh.wast',
+    'eh-gc.wast',
     'exception-handling.wast',
+    'global-effects.wast',
+    'local-subtyping.wast',
+    'renamings.wat',
```

Comment:
These three tests are not strictly wasm-EH tests, though perhaps it is fine to disable them for now as they are not terribly important. Maybe put them at the end with a separate comment like `These contain parts with new EH. TODO: split those parts out if we see that fuzzing new EH is far out`. (but my hope is that it is not far out).